concrete/src/Attribute/ObjectTrait.php

```diff
@@ -104,6 +104,11 @@ public function setAttribute($ak, $value)
                  */
                 $value = $controller->createAttributeValue($value);
             }
+        } else {
```

Comment:
Looking only at the code I think technically since the parent will return a clone we can just get rid of this `else` and the related `if` statement all together FWIW